collect_app/src/main/java/org/odk/collect/android/utilities/ResetUtility.java

```diff
@@ -35,7 +35,11 @@
 public class ResetUtility {
 
     private List<Integer> failedResetActions;
-    private final StoragePathProvider storagePathProvider = new StoragePathProvider();
+    private final StoragePathProvider storagePathProvider;
+
+    public ResetUtility() {
+        storagePathProvider = new StoragePathProvider();
```

Comment:
What is the advantage of this change?

Reply:
oooo nice catch. That's a mistake. I think I'd looked at passing one in for some reason but abandoned the idea. I'll revert that.